v2/internal/pkg/operator/new.go

```diff
@@ -14,5 +14,15 @@ func New(log clog.PluggableLoggerInterface,
 	mirror mirror.MirrorInterface,
 	manifest manifest.ManifestInterface,
 ) CollectorInterface {
-	return &LocalStorageCollector{Log: log, LogsDir: logsDir, Config: config, Opts: opts, Mirror: mirror, Manifest: manifest, LocalStorageFQDN: opts.LocalStorageFQDN}
+	return &LocalStorageCollector{OperatorCollector{Log: log, LogsDir: logsDir, Config: config, Opts: opts, Mirror: mirror, Manifest: manifest, LocalStorageFQDN: opts.LocalStorageFQDN, ctlgHandler: catalogHandler{Log: log}}}
```

Comment:
nit: it could be possible to keep only one new if at this point we had access to the flag --alpha-ctlg-filter. Do we have access to this flag here through the opts?